tests/installation/partitioning_filesystem.pm

```diff
@@ -28,11 +28,6 @@ sub run {
         assert_screen([qw(select-hard-disks partition-scheme)]);
         select_first_hard_disk if match_has_tag 'select-hard-disks';
     }
-
-    if (get_var('PARTITIONING_WARNINGS')) {
```

Comment:
osd seems to be down atm, so I will just ask here so we don't forget =) do we have this test executed for older versions where `PARTITIONING_WARNINGS` is still used? Also, we need a new schedule for TW, as it currently uses main.pm schedule: https://openqa.opensuse.org/tests/1498499

Reply:
I found only very old jobs on older versions, that use this `PARTITIONING_WARNINGS`. I strongly believe that this variable should be removed from our tests as this is a hack. So, even if I missed something and some test will be broken because of that, I prefer to fix it instead of trying to keep this variable and add even more complexity to the test logic.

Thanks for the info about TW. Didn't notice that. I thought we already moved all out test suites to the yaml:)